cmd/notary/tuf.go

```diff
@@ -464,11 +466,19 @@ func readRootKey(rootKeyFile string, retriever notary.PassRetriever) (data.Priva
 	if err != nil {
 		return nil, fmt.Errorf("Error reading input root key file: %v", err)
 	}
+	isEncrypted := true
 	if err = cryptoservice.CheckRootKeyIsEncrypted(pemBytes); err != nil {
-		return nil, err
+		if role == data.CanonicalRootRole {
+			return nil, err
+		}
+		isEncrypted = false
+	}
+	var privKey data.PrivateKey
+	if isEncrypted {
+		privKey, _, err = trustmanager.GetPasswdDecryptBytes(retriever, pemBytes, "", data.CanonicalRootRole)
```

Comment:
Oops, sorry I missed this question before for some reason!  Hm... I thought we encrypted all the keys on disk (hence it asks for your passphrase when a targets key is generated).  For instance, in `cmd/notary/keys.go` above, `readKey` is called when rotating, which could technically be for any role.